tests/examples/logging/playground/printstuff/run

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+exec 2>&1
```

Comment:
This is the "standard" way to get stderr of a service to multilog because multilog only accepts stdout.